tests/framework/microvm.py

```diff
@@ -460,9 +469,17 @@ def pin_api(self, cpu_id: int):
             return True
         return False
 
+    def pin_vhost_user_blk_backend(self, socket, cpu_id: int):
```

Comment:
Could this move to the `VhostUserBlkBackend` class?

Reply:
but there should still be a method in `Microvm` to wrap it?

Reply:
Maybe we can expose `vhost_user_blk_backends` as a member? So pinning will be: `vm.vu_block_backends[socket].pin(...)` So `vm` will own the backends, but we can access them anyway to configure them.

Reply:
I don't particularly like this for 2 reasons:
- this will strengthen utils_drive.py dependency on microvm.py
- this may invite accessing the member in other places and make it harder to keep track

What is it specifically that don't you like about `pin_vhost_user_blk_backend()` being in the microvm.py?

Reply:
My 2c is we should avoid provide wrapper methods that don't add some convenience/value. In this case it just forwards the function call. Ditto for resize_vhost_user_drive and kill_vhost_user_blk_backend.